arlo-client/src/components/CreateAudit.tsx

```diff
@@ -23,11 +24,21 @@ const Wrapper = styled.div`
 const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
   const [loading, setLoading] = useState(false)
   const onClick = async () => {
-    setLoading(true)
-    const { electionId } = await api('/election/new', {
-      method: 'POST',
-    })
-    history.push(`/election/${electionId}`)
+    try {
```

Comment:
If you were to update the type of `toaster` I as suggested, you could do this here:

```ts
    try {
      setLoading(true)
      const response = await api<{ electionId: string }>('/election/new', {
        method: 'POST',
      })
      if (toaster(response)) {
        return
      }
      const { electionId } = response
      history.push(`/election/${electionId}`)
    } catch (err) {
      toast.error(err.message)
    }
```